instrumented/integration/src/androidTest/kotlin/com/datadog/android/sdk/integration/sessionreplay/SrTest.kt

```diff
@@ -172,9 +254,12 @@ internal abstract class SrTest<R : Activity, T : MockServerActivityTestRule<R>>
 
     companion object {
         internal val INITIAL_WAIT_MS = TimeUnit.SECONDS.toMillis(60)
+        private const val UI_THREAD_DELAY_IN_MS = 1000L
+        private const val PAYLOAD_UPDATE_REQUEST = "updateSrPayloads"
```

Comment:
can also name it `updateSessionReplayPayloads`. IMO having `SessionReplay` not shortened will simplify text search, because searching only for `sr` may give too much noise.